lib/ioki/model/operator/product_bulletin.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Ioki
+  module Model
+    module Operator
+      class ProductBulletin < Base
+        attribute :type,
+                  on:   :read,
+                  type: :string
+
+        attribute :id,
+                  on:   :read,
+                  type: :string
+
+        attribute :severity,
+                  on:   :read,
+                  type: :string
+
+        attribute :title,
+                  on:   :read,
+                  type: :string
+
+        attribute :description,
+                  on:   :read,
+                  type: :string
+
+        attribute :affected_items,
+                  on:         :read,
+                  type:       :array,
```

Comment:
This is probably intentional but it startled me for a moment:

When there are no affected items like for `not_live` the API returns `null`

```json
  {
        "id": "not_live",
        "severity": "danger",
        "title": "Product NOT live",
        "description": "This product is in \"planned\" mode. If you operate a service, make sure to set this product to \"live\" mode, as non live products skip safety checks and critical errors might not get detected.",
        "affected_items": null,
        "type": "product_bulletin"
      }
```

When reading the attribute definition above I would expect a conversion, however its `nil`

```ruby
{:type=>"product_bulletin",
 :id=>"not_live",
 :severity=>"danger",
 :title=>"Produkt NICHT Live",
 :description=>
  "Dieses Produkt befindet sich im Modus \"planned\". Wenn Sie einen Dienst betreiben, stellen Sie sicher, dass dieses Produkt in den \"Live\"-Modus versetzt wird, da Produkte, die nicht Live sind, Sicherheitsprüfungen überspringen und kritische Fehler möglicherweise nicht erkannt werden.",
 :affected_items=>nil}
```

Reply:
You mean, you'd expect an empty hash for `affected_items`?

Reply:
An empty array, yes

```
{:type=>"product_bulletin", :id=>"not_live", :severity=>"danger", :title=>"Produkt NICHT Live", :description=> "Dieses Produkt befindet sich im Modus \"planned\". Wenn Sie einen Dienst betreiben, stellen Sie sicher, dass dieses Produkt in den \"Live\"-Modus versetzt wird, da Produkte, die nicht Live sind, Sicherheitsprüfungen überspringen und kritische Fehler möglicherweise nicht erkannt werden.", :affected_items=>[]}
```

Reply:
Good point!
The issue can be found in the `ProductBulletin` class in triebwerk. The default value for affected_items is `nil` 😬 . I will make a follow up PR in triebwerk. 👍

Reply:
Let's change it to an empty array, that's more consistent.